pkg/chunk/storage/factory.go

```diff
@@ -128,7 +128,7 @@ func NewStore(cfg Config, storeCfg chunk.StoreConfig, schemaCfg chunk.SchemaConf
 	}
 
 	chunkCacheCfg := storeCfg.ChunkCacheConfig
-	chunkCacheCfg.Prefix = "chunks"
+	chunkCacheCfg.Prefix = "chunks."
```

Comment:
There few implications in this change and I'm not sure the change here is correct. For example, this prefix is also used by fifocache `Instrument(cfg.Prefix+"fifocache", cache)` which then leads to `method := i.name + ".store"`. Adding the dot will cause the method name to be `chunks..store`.

Could you take a deeper look at the effect of this change, please?